OpenAI/Packages/com.openai.unity/Runtime/Common/Content.cs

```diff
@@ -5,7 +5,7 @@
 using UnityEngine;
 using UnityEngine.Scripting;
 
-namespace OpenAI.Chat
+namespace OpenAI
 {
     [Preserve]
     public sealed class Content
```

Comment:
@StephenHodgson I've noticed that the Content isn't parsed when using `MessageItem.PrintContents()`.
It seems that it's because of the `Content` object is now an array, which can contain an `annountations: []` array.
Have you noticed this behavior too?

Example MessageResponse:
```
{
  "object": "list",
  "data": [
    {
      "role": "assistant",
      "content": [
        {
          "type": "text",
          "text": {
            "value": "...",
            "annotations": []
          }
        }
      ],
      "attachments": [],
      "metadata": {}
    }
  ]
}
```

Reply:
I'm aware. I have not ported this change from the OpenAI-DotNet repo yet.